### ISSUE TITLE
Are there any pytorch resnet-101 imagenet pretrained network? How to download?

### ISSUE THREAD

**Issue by ZhangGongjie** [multimodallearning/pytorch-mask-rcnn#44]
Thank you so much
